### ISSUE TITLE
cpu inference speed

### ISSUE THREAD

**Issue by Zalways** [Z-Xiong/OSTrack-mnn#5]
hello! thank you for your fantastic work! i tried your c++ code, and inferenced on my cpu, but the inference speed is too slow, with the 384-ep300 model, FPS only 1.3, and with the ostrack-bit8 model FPS only 3.

**Comment by Zalways** [comment]
i tried to deploy the models in QT.

**Comment by Z-Xiong** [comment]
Yes, the inference time is too slow when using cpu. If you want to get an efficient track model, you can try this: https://github.com/PinataFarms/FEARTracker, which is pure CNN.
